mautic/mautic
# Fixed cs-fixer in GitHub actions

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features"
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      |
| Related developer documentation PR URL |
| Issue(s) addressed                     | 

#### Description:
This PR fixes `CS Fixer` GitHub Action. There were several issues:
1. #10023 was failing with `[Symfony\Component\Console\Exception\RuntimeException] The "--git" option does not exist.`
2. When there was no `*.php` modified file,  `bin/php-cs-fixer fix ...` checked all PHP files in the project.
3. As there was `git diff "HEAD~..HEAD"`, `bin/php-cs-fixer fix ...` checked only PHP files that had been changed in the very last commit.
4. Now when there is no changed  `*.php` file, `bin/php-cs-fixer fix ...` is not executed at all.

#### Steps to test this PR:
1. Try changing several PHP files so they break CS rules => GitHub CS Fixer action should fail.
2. Try changing several non-PHP files while not changing any PHP file =>  GitHub CS Fixer should not execute `bin/php-cs-fixer fix ...` command at all.
